sdk/eventgrid/microsoft-azure-eventgrid/pom.xml

```diff
@@ -110,7 +110,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>2.8</version> <!-- {x-version-update;eventgrid_org.apache.maven.plugins:maven-javadoc-plugin;external_dependency} -->
+                <version>3.1.1</version> <!-- {x-version-update;org.apache.maven.plugins:maven-javadoc-plugin;external_dependency} -->
```

Comment:
The POM file exception for eventgrid_org.apache.maven.plugins:maven-javadoc-plugin was removed and all tooling should have been at the same version. The script detected and report this.